src/System.Net.Ping/src/System.Net.Ping.csproj

```diff
@@ -88,8 +91,13 @@
       <Link>Common\Interop\Unix\System.Native\Interop.SocketAddress.cs</Link>
     </Compile>
   </ItemGroup>
-  <ItemGroup Condition=" '$(TargetsWindows)' == 'true'">
+  <ItemGroup Condition="'$(TargetsWindows)' == 'true' And '$(TargetGroup)' != 'uap'">
     <Compile Include="System\Net\NetworkInformation\Ping.Windows.cs" />
+  </ItemGroup>
+  <ItemGroup Condition="'$(TargetGroup)' == 'uap'">
```

Comment:
I will prefer 
`<ItemGroup Condition="'$(TargetsWindows)' == 'true' And '$(TargetGroup)' == 'uap'">` here, because I have seen this pattern in some other S.N .csprj file.

@davidsh @CIPop please correct me if I'm wrong.